### PR TITLE
Move cat: `Creating user-centred content`

### PR DESCRIPTION
**Pull request by shad0wMe0w**
- Move entire category
- Refactor hyperlinks to content within this category
- Add category card to homepage
- Add missing link on the category page.

### CODE REVIEW

File: content/homepage/categories.md [hunk 1]
```diff
@@ -21,4 +21,8 @@ cards:
     text: A content strategy guides the creation, delivery and governance of useful, usable content.
     link: '/content-strategy/'
     background: '#1B7991'
+  - headline: Creating user-centred content
```
Comment (joolswood) [inline content/homepage/categories.md]: Maybe use a non-breaking hyphen? It's kinda awkward with 1 word on one line, but breaking after the hyphen looks awkward too? ¯\_(ツ)_/¯

File: content/creating-user-centred-content/content-design/methods.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Good content design begins with user research. This is then translated into simp
 
 A good content designer always asks what the user need is and how to meet this need. Be careful not to exclude user groups or assume what services users might need. Always think about [inclusivity and diversity](/user-research/identifying-users-needs/#make-your-research-inclusive).
 
-To create user-centred content you need to [research your users](/creating-user-centred-content/using-existing-research), which includes [using existing research](/content-strategy/creating-user-centred-content/using-existing-research/).
+To create user-centred content you need to [research your users](/creating-user-centred-content/using-existing-research), which includes [using existing research](/creating-user-centred-content/using-existing-research/).
```
Comment (joolswood) [inline content/creating-user-centred-content/content-design/methods.md]: These links are broken. Suggest also `research your users` should be `research with your users`. Liz OKed this change in the Google Doc.

**Comment by shad0wMe0w** [comment]
☝️ changes made.

**Comment by joolswood** [comment]
LGTM

**Pull request merged**